manager/apis/app/v1beta1/fybrikapplication_webhook_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1beta1
+
```

Comment:
Same comment - the parallel file for v1alpha1 is not needed.

Reply:
The same answer.

Reply:
Ok, I missed that.